core/app/models/spree/order.rb

```diff
@@ -21,7 +21,6 @@ class Order < Spree::Base
 
     alias display_ship_total display_shipment_total
     alias_attribute :ship_total, :shipment_total
-    alias_attribute :token, :guest_token # FIXME: guest_token should be renamed to just token
```

Comment:
@kushniryb let's add an alias `token -> guest_token` with a deprecation notice, this will allow people to gracefully update and migrate

Reply:
What exactly do you mean by that?

Reply:
@kushniryb yes